### DISCUSSION TITLE
Выделение области на карте

### DISCUSSION THREAD

**Discussion by dongaber** [yandex-maps-unofficial/vue-yandex-maps#128] [Q&A (2.x)] [answered]
Хотел бы поинтересоваться, как сделать функционал выделения области, реализация есть в документации yandex api v2.1:
https://yandex.ru/dev/maps/jsbox/2.1/polygon_custom/

Я использую 2 версию вашей библиотеки и не понимаю как сделать выделение.

**Comment by daniluk4000** [comment, marked as answer]
Приветствую. Версия 2.0 использует Карты 3.0, где аналогичного примера с рисованием нет. 

Полагаю, реализовывать его вам придётся самостоятельно. Как раз во вчерашнем обновлении была добавлена возможность добавлять в карту `<yandex-map-entity>` со своим содержимым, куда можно вставить тот же канвас, а далее на основе пользовательских данных и кода из старого примера добавить `<yandex-map-feature>` с нужными settings.

**Comment by daniluk4000** [comment]
Если всё таки дойдёте до реализации, можете сделать PR и добавить новый пример, где покажете, как этого можно добиться в новой версии, чем поможете другим пользователям. Или просто скинуть сюда пример компонента/кода.

**Comment by dongaber** [comment]
Хорошо, спасибо за направление, сегодня займусь реализацией, если что то получится сделаю PR с примером

**Reply by Misha6787** [reply]
Добрый день, у меня похожая задача, хотел бы узнать удалось ли вам реализовать такой функционал, и возможно ли посмотреть на код этой реализаций? И если пример выгружен в PR, я просто не знаю что значит PR.... извините

**Reply by daniluk4000** [reply]
PR - Pull Request) 

Пример не выгружался

**Reply by Misha6787** [reply]
Понял, хотел на другую реализацию взглянуть, у меня как то вышло через использование внешнего canvas, но пока ищу вариант как получать маркеры в этой области, это по сути единственная оставшиеся проблема
